main.py

```diff
@@ -1,10 +1,47 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+    for nickname, players_info in data.items():
+        race_obj, created = Race.objects.get_or_create(
+            name=players_info["race"]["name"],
+            defaults={
+                "description": players_info["race"]["description"]
+            }
+        )
+        skills = players_info["race"]["skills"]
+        for skill in skills:
+            skill_obj, created = Skill.objects.get_or_create(
+                name=skill["name"],
+                defaults={
+                    "bonus": skill["bonus"],
```

Comment:
Ensure that the 'bonus' value from the JSON data is compatible with the 'bonus' field type in the Skill model. If the field type is changed to IntegerField or FloatField, make sure the JSON data provides numeric values.